tests/unit/environments/BinaryLabelData.h

```diff
@@ -0,0 +1,86 @@
+#ifndef BINARY_LABEL_DATA_HPP
+#define BINARY_LABEL_DATA_HPP
+
+#ifdef HAVE_LAPACK
+
+#include <shogun/features/DataGenerator.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/labels/BinaryLabels.h>
+
+using namespace shogun;
+#ifdef USE_GPL_SHOGUN
```

Comment:
Because karlnapf said "license missing" in https://github.com/shogun-toolbox/shogun/pull/3711#discussion_r107633412. So I just assume all the file must been guarded by GPL. Or I should add license in the other way